### ISSUE TITLE
Prismarine Ring causes Player on Server to Disconnect - java.lang.NoClassDefFoundError

### ISSUE THREAD

**Issue by TigerWalts** [WinDanesz/AncientSpellcraft#11]
**Version**: `1.1.1`

The `ASArtefactHandler.onPlayerTickEvent()` method tries to access the `net/minecraft/client/Minecraft` class when the player has a Prismarine Ring and is burning:

https://github.com/WinDanesz/AncientSpellcraft/blob/d127128e015c1d4b18f9346cd6994e4dc218b99e/src/main/java/com/windanesz/ancientspellcraft/handler/ASArtefactHandler.java#L538

This causes a non-fatal Exception that terminates the player's connection:

```
Server thread/ERROR Exception caught during firing event net.minecraftforge.fml.common.gameevent.TickEvent$PlayerTickEvent@751a4e8:
java.lang.NoClassDefFoundError: net/minecraft/client/Minecraft
at com.windanesz.ancientspellcraft.handler.ASArtefactHandler.onPlayerTickEvent(ASArtefactHandler.java:519) ~[ASArtefactHandler.class:?]
```

Current workaround: Disabling the ring in the `ebwizardry.cfg` should stop this code being executed as `electroblob.wizardry.item.ItemArtefact.getActiveArtefacts()` will not return a disabled item.

**Comment by WinDanesz** [comment]
Thanks for the detailed error report, will be fixed in the next patch soon!

**Comment by WinDanesz** [comment]
Fixed in 1.1.2
